backend/api/Services/ReturnToHomeService.cs

```diff
@@ -5,7 +5,10 @@ namespace Api.Services
 {
     public interface IReturnToHomeService
     {
-        public Task<MissionRun?> ScheduleReturnToHomeMissionRunIfNotAlreadyScheduled(Robot robot);
+        public Task<MissionRun?> ScheduleReturnToHomeMissionRunIfNotAlreadyScheduled(
+            Robot robot,
+            bool shouldTriggerMissionCreatedEvent = false
```

Comment:
does this need to be configurable or can it be set to true for all calls for this function?